computer-vision/README.md

```diff
@@ -0,0 +1,21 @@
+# CVRA 2019 Computer vision
+
+
+Requirements:
+
+- Detect weathervane orientation using its Aruco marker (6cm 4x4 ArUco Nb 17 at [1500,0])
+- Detect the random reefs configuration
+- Send this to the master firmware
+
+## Camera calibration
+
+Follow the OpenCV tutorial: [Camera calibration With OpenCV](https://docs.opencv.org/2.4/doc/tutorials/calib3d/camera_calibration/camera_calibration.html).
+You may also read [Camera Calibration and 3D Reconstruction](https://docs.opencv.org/2.4/modules/calib3d/doc/camera_calibration_and_3d_reconstruction.html).
+
+Take some picture with the script `tools/grap_image.py` and then run `tools/calibrate.py` to get the camera matrix and distortion coefficient vector.
+
+Note: Fisheye camera need different procedure using [fisheye::calibrate](https://docs.opencv.org/2.4/modules/calib3d/doc/camera_calibration_and_3d_reconstruction.html?highlight=calibratecamera#fisheye-calibrate)
+
+## ArUco tracking
```

Comment:
Add a section about how to run it?

Reply:
We will. But it is not finished yet, so for now there is no "how to run".

Reply:
Not even a "how to dev"?

Reply:
No how to dev instructions? How do you test this code?